### ISSUE TITLE
[BUG] Logrotate config has incorrect permissions

### ISSUE THREAD

**Issue by ryangsteele** [go-graphite/carbonapi#579]
**Describe the bug**
Debian package installs carbonapi's logrotate config with bad file mode

**CarbonAPI Version**
0.14.1-1 (reproducible with latest stable package from packagecloud)

**Logs**
```
root@graphite:~# /etc/cron.daily/logrotate
error: Ignoring carbonapi because of bad file mode - must be 0644 or 0444.

root@graphite:~# ls -alt /etc/logrotate.d/carbonapi
-rw-rw-r-- 1 root root 137 Aug 19  2020 /etc/logrotate.d/carbonapi
```

**CarbonAPI Configuration:**
N/A 

**Simplified query (if applicable)**
N/A

**Backend metric retention and aggregation schemas**
N/A

**Backend response (if possible)**
N/A

**Additional context**
Once perms are changed to 0644, running `/etc/cron.daily/logrotate` properly adds carbonapi to `/var/lib/logrotate/status` and will rotate the carbonapi logs successfully going forward.

**Comment by Civil** [comment]
Should be fixed in packages starting from `carbonapi-0.15.2~57+sha.gbb33f2`

**Comment by richarson** [comment]
I see this error again:

/etc/cron.daily/logrotate:
error: Ignoring carbonapi because of bad file mode - must be 0644 or 0444.

[root@server ~] # rpm -q carbonapi
carbonapi-0.15.4~1-1.x86_64
